packages/renderer/src/lib/ui/KubernetesCheckConnection.svelte

```diff
@@ -0,0 +1,27 @@
+<script lang="ts">
+import { faRefresh } from '@fortawesome/free-solid-svg-icons';
+import { Button } from '@podman-desktop/ui-svelte';
+
+import {
+  kubernetesContextsCheckingStateDelayed,
+  kubernetesCurrentContextState,
+} from '/@/stores/kubernetes-contexts-state';
+
+import { kubernetesContexts } from '../../stores/kubernetes-contexts';
+
+let currentContextName: string | undefined;
+$: currentContextName = $kubernetesContexts.find(c => c.currentContext)?.name;
```

Comment:
suggestion: use new svelte5 runes

```suggestion
let currentContextName = $derived($kubernetesContexts.find(c => c.currentContext)?.name)
```

Reply:
Will change to svelte5 in a follow-up PR

Reply:
you only have to click on the button :-)

Reply:
But it would be a mix between svelte 4 and 5, no?

I would also like to change the `$:`s by `$state`s

But that's not a big deal. I'll make the changes here

Reply:
done

Reply:
thanks